livekit-server/values.yaml

```diff
@@ -43,6 +47,15 @@ livekit:
   # room: <default settings for new rooms>
   # region: <set for multi-region deploys>
 
+keyFile:
+  enabled: false
+  # Use a pre existing secret, useful to combine with external secret managers
+  # as GCP External Secrets or Hashicorp Vault
+  existingSecret: ""
+  keys:
```

Comment:
would be good to add a comment indicating keys here are stored in Secret and not in the config file.

Reply:
Done!